src/Entity/ShopBillingDataInterface.php

```diff
@@ -13,19 +13,56 @@
 
 namespace Sylius\RefundPlugin\Entity;
 
-interface ShopBillingDataInterface
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface ShopBillingDataInterface extends ResourceInterface
 {
-    public function id(): ?int;
+    public function getId();
+
+    public function setId($id): void;
```

Comment:
We won't need `setId` on the interface - see https://github.com/Sylius/RefundPlugin/pull/315/files#r653493427.